tests/electron/modules/compliments_spec.js

```diff
@@ -88,7 +88,7 @@ describe("Compliments module", () => {
 			});
 		});
 		describe("get updated list from remote file", () => {
-			it("shows 'test in morning' as test time set to 10am", async () => {
+			it("shows 'test in morning'", async () => {
```

Comment:
why get rid of the text here?

Reply:
because the test is loading the file, not what time it is..  the test for the text is just to confirm it was loaded.